adapter/config/parser.go

```diff
@@ -229,6 +236,26 @@ func (config *Config) resolveDeprecatedProperties() {
 
 }
 
+func (config *Config) resolveJWTGeneratorConfig() error {
+	KeyPairs := config.Enforcer.JwtGenerator.Keypair
+	signingCount := 0
+	for i, keypair := range KeyPairs {
+		if keypair.UseForSigning {
+			signingCount++
+		} else {
+			// Removing non signing private key paths from config
```

Comment:
If it is signing key, lets validate if the both private key and public key is there and that would be sufficient. We do not need to modify the user input here.

Reply:
we're only modifying non signing key paths here, so they don't go on to the enforcer, I can add a check for the signing key paths , shouldnt this stay though?